CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+How to contribute
+=================
+
+Thanks for your interest in contributing to Kinto!
+
+## Reporting Bugs
+
+Report bugs at https://github.com/Kinto/kinto-http.py/issues/new
+
+If you are reporting a bug, please include:
+
+ - Any details about your local setup that might be helpful in troubleshooting.
+ - Detailed steps to reproduce the bug.
```

Comment:
- Detailed steps to reproduce the bug or even a PR with a failing tests if you can.